### ISSUE TITLE
could not branch Ubuntu repository: bzr: ERROR: Not a branch

### ISSUE THREAD

**Issue by odg0318** [quay/clair#563]
**Description of Problem**:
It is unavailable to update Ubuntu repository as the following error message.
```
{"Event":"could not branch Ubuntu repository","Level":"error","Location":"ubuntu.go:177","Time":"2018-06-29 05:52:09.996412","error":"exit status 3","output":"bzr: ERROR: Not a branch: \"https://launchpad.net/ubuntu-cve-tracker/\".\n"}
```

**Expected Outcome**:
Successfully updated.

**Actual Outcome**:
Canceled with some errors.

**Environment**:
- Clair version/image: 2.0.3
- Clair client name/version: 2.0.3
- Host OS: CentOS7
- Kernel (e.g. `uname -a`): 3.10.0
- Kubernetes version (use `kubectl version`): X
- Helm version (use `helm version`): X
- Network/Firewall setup: Through proxy to access to repository.

**Additional Question**
When will new version be released including enabling repository feature?

**Comment by SysAdmLS** [comment]
The tracker has been converted to git and is now available from https://code.launchpad.net/~ubuntu-security/ubuntu-cve-tracker/+git/ubuntu-cve-tracker 

See: https://bugs.launchpad.net/ubuntu-cve-tracker/+bug/1771169

**Comment by odg0318** [comment]
@SysAdmLS Thank you for your reply. How can I fix this issue?

**Comment by SysAdmLS** [comment]
@odg0318 Wait for this pull : https://github.com/coreos/clair/pull/562
to merge.

Or disable the Ubuntu CVE Tracker in your config.yml
Edit: Disabling the Ubuntu CVE Tracker does not fix this issue.

**Comment by odg0318** [comment]
@SysAdmLS Additionally, disabling ubuntu  is not released yet on latest version.

**Comment by SysAdmLS** [comment]
Well there is this in the config-yml

  updater:
    # Frequency the database will be updated with vulnerabilities from the default data sources
    # The value 0 disables the updater entirely.
    interval: 2h
    enabledupdaters: 
      - debian
      - ubuntu
      - rhel
      - oracle
      - alpine

But as is said it still tries to fetch from the Ubuntu Tracker. Deleting the Ubuntu line only results in Clair not adding the Ubuntu CVE's to the Database, not disabling the whole fetching process of its CVE's.

**Comment by danmx** [comment]
@SysAdmLS it looks like a bug

**Comment by jzelinskie** [comment]
I'm going to be testing and cutting a new release soon that uses Ubuntu's git repository. Bazaar will no longer be required for Clair and this error will not occur again.

Fixed in #568
